src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
```

Comment:
Prefer `const` over `let` for the `result` variable if you are not planning to reassign it, which is the case in this function after refactoring based on the checklist items 3 and 5.